src/components/Vault/VaultTable.tsx

```diff
@@ -1,19 +1,19 @@
+import dynamic from "next/dynamic"
 import { FC } from "react"
 
-import { useVaultAddresses } from "@/hooks/data"
 import { VaultProps } from "@/hooks/types"
 
-import Spinner from "@/components/Spinner"
-import VaultRow from "@/components/Vault/VaultRow"
 import {
   VaultTableHeader,
   VaultTableRow,
 } from "@/components/Vault/VaultTableNode"
 
+const VaultTableBody = dynamic(
```

Comment:
Moving the dynamic import down to the VaultBody component to reduce layout shift a bit more. Headers will be on the page when it is loaded by the client.